stats/config/charts.json

```diff
@@ -65,96 +65,96 @@
         },
         "accounts_growth": {
             "title": "Accounts growth",
-            "description": "Cumulative accounts number per period"
+            "description": "Cumulative accounts number"
         },
         "new_accounts": {
             "title": "New accounts",
-            "description": "New accounts number per day"
+            "description": "New accounts number per period"
         },
         "average_txn_fee": {
             "title": "Average transaction fee",
-            "description": "The average amount in {{native_coin_symbol}} spent per transaction",
+            "description": "The average amount in {{native_coin_symbol}} spent for transaction per period",
             "units": "{{native_coin_symbol}}"
         },
         "txns_fee": {
             "title": "Transactions fees",
-            "description": "Amount of tokens paid as fees",
+            "description": "Amount of tokens paid as fees per period",
             "units": "{{native_coin_symbol}}"
         },
         "new_txns": {
             "title": "New transactions",
-            "description": "New transactions number"
+            "description": "New transactions number per period"
         },
         "txns_growth": {
             "title": "Transactions growth",
             "description": "Cumulative transactions number"
         },
         "txns_success_rate": {
             "title": "Transactions success rate",
-            "description": "Successful transactions rate per day"
+            "description": "Successful transactions rate per period"
         },
         "new_blocks": {
             "title": "New blocks",
-            "description": "New blocks number"
+            "description": "New blocks number per period"
         },
         "average_block_size": {
             "title": "Average block size",
-            "description": "Average size of blocks in bytes",
+            "description": "Average size of blocks per period",
             "units": "Bytes"
         },
         "average_block_rewards": {
             "title": "Average block rewards",
-            "description": "Average amount of distributed reward in tokens per day",
+            "description": "Average amount of distributed rewards in tokens per period",
```

Comment:
Average {{native_coin_symbol}} reward per block

Reply:
When calculating we take an average value of all rewards during the period (e.g. day). It's possible to have multiple rewards for the same block (for some chains), so I think something like "Average amount of reward" would be more accurate. What do you think?